guides/v2.1/cloud/live/sens-data-initial.md

```diff
@@ -104,14 +105,17 @@ The following snippet from `config.local.php` shows an example of changing the d
 
      ... more ...
 </pre>
+```
 
 ## Push and deploy config.local.php to environments {#deploy}
 
 Now that you've created `config.local.php` and transferred it to your local workstation, commit it to Git and push it to your environments. This procedure corresponds to step 3 and 4 in the [recommended procedure]({{ page.baseurl }}/cloud/live/sens-data-over.html#cloud-config-specific-recomm).
 
 The following command adds, commits, and pushes to master:
 
-	git add app/etc/config.local.php && git commit -m "Add system-specific configuration" && git push origin master
+```bash
```

Comment:
Remove blank line below and on the others.

Reply:
**done**